internal/db/schema/migrations/oss/postgres/37/01_set_unlimited_connections_limit.up.sql

```diff
@@ -0,0 +1,7 @@
+begin;
+
```

Comment:
Can you add a comment to note that this changes the default that was set in `0/41_targets.up.sql` and add a comment to `0/41_targets.up.sql` to note that the default changed in this migration.

Reply:
Done